contingency/contingency-api/src/main/java/com/powsybl/contingency/tasks/LineTripping.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.contingency.tasks;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.Network;
+
+/**
+ * @author Mathieu Bague <mathieu.bague@rte-france.com>
+ */
+public class LineTripping extends BranchTripping {
```

Comment:
Do we really need to subclass BranchTripping as we do not add anything from base class (just more precise error message) ?